xrpl/models/transactions/permissioned_domain_delete.py

```diff
@@ -0,0 +1,41 @@
+"""Model for PermissionedDomainDelete transaction type."""
+
+import re
+from dataclasses import dataclass, field
+from typing import Dict, Final, Pattern
+
+from typing_extensions import Self
+
+from xrpl.models.required import REQUIRED
+from xrpl.models.transactions.transaction import Transaction
+from xrpl.models.transactions.types import TransactionType
+from xrpl.models.utils import KW_ONLY_DATACLASS, require_kwargs_on_init
+
+_DOMAIN_ID_REGEX: Final[Pattern[str]] = re.compile("[A-F0-9]{64}")
+
+
+@require_kwargs_on_init
+@dataclass(frozen=True, **KW_ONLY_DATACLASS)
+class PermissionedDomainDelete(Transaction):
+    """This transaction deletes a PermissionedDomain object."""
+
+    domain_id: str = REQUIRED  # type: ignore
+    """The domain to delete."""
+
+    transaction_type: TransactionType = field(
+        default=TransactionType.PERMISSIONED_DOMAIN_DELETE,
+        init=False,
+    )
+    """The transaction type (PermissionedDomainDelete)."""
+
+    def _get_errors(self: Self) -> Dict[str, str]:
+        errors = super()._get_errors()
+
+        if len(self.domain_id) != 64:
```

Comment:
We should use a variable for this.
```suggestion
        if len(self.domain_id) != DOMAIN_ID_LENGTH:
```

Reply:
7e9fdfb3